packages/dara-components/changelog.md

```diff
@@ -2,9 +2,13 @@
 title: Changelog
 ---
 
-## NEXT
+## 1.8.6
 
 -   Fixed an issue where `Chat` button looked off centre on different base font size apps.
+-   Removed margin from `Chat` component
```

Comment:
We don't need this one because I was setting margin to 0 on dara side until we released the UI so it doesn't really change the visual

Reply:
Good catch, thanks